### PR TITLE
Enable NumberFormatInfo netstandard 1.7 APIs

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @jkotas @weshaggard @danmosemsft

File: src/mscorlib/corefx/System/Globalization/DigitShapes.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+// The enumeration constants used in NumberFormatInfo.DigitSubstitution.
+//
+namespace System.Globalization
+{
+    [Serializable]
+    [System.Runtime.InteropServices.ComVisible(true)]
```
Comment (jkotas) [inline src/mscorlib/corefx/System/Globalization/DigitShapes.cs]: ComVisible is not necessary

File: src/mscorlib/corefx/System/Globalization/NumberFormatInfo.cs [hunk 2]
```diff
@@ -748,6 +808,30 @@ public String PerMilleSymbol
             }
         }
 
+        [System.Runtime.InteropServices.ComVisible(false)]
```
Comment (jkotas) [inline src/mscorlib/corefx/System/Globalization/NumberFormatInfo.cs]: A few more....

**Comment by tarekgh** [comment]
test Windows_NT arm Cross Checked Build please

**Comment by tarekgh** [comment]
test Linux ARM Emulator Cross Debug Build please

**Pull request merged**